### PR TITLE
Add Search spinner to QA InputLookup

### LINKED ISSUE #2615: Add spinner to QA lookup on resource
There should be a spinner when waiting for a fetch from QA when editing a resource.

Use the same pattern as implemented on https://github.com/LD4P/sinopia_editor/pull/2567 for Sinopia search

### PR DESCRIPTION
**Pull request by jgreben**
## Why was this change made?
Fixes #2615. Adds search spinner and ability to clear search results tor QA Authority InputLookup.

## How was this change tested?
Will be tested as part of #2257 as soon as that is unblocked

### CODE REVIEW

File: src/components/editor/property/LookupWithMultipleAuthorities.jsx [hunk 1]
```diff
@@ -41,6 +41,8 @@ const LookupWithMultipleAuthorities = (props) => {
     return Object.values(authorities)
   }, [props.propertyTemplate.authorities])
 
+  const clearSearchResults = useCallback(() => dispatch(clearSearchResultsAction('resource')), [dispatch])
```
Comment (justinlittman) [inline src/components/editor/property/LookupWithMultipleAuthorities.jsx:44]: Doesn't this clear the search results of the main search, not lookup?

**Review by justinlittman — CHANGES_REQUESTED** [review]

**Comment by justinlittman** [comment]
@jgreben You might want to hold off on this. Significant refactor of InputLookup forthcoming.

File: src/components/editor/property/LookupWithMultipleAuthorities.jsx (hunk 1 above)
Comment (jgreben) [reply src/components/editor/property/LookupWithMultipleAuthorities.jsx:44]: It appears to work on both.

**Comment by jermnelson** [comment]
@jgreben can you rebase this PR? Would like to get this PR merged before doing the final `3.5.0` tagged release for this work-cycle.

**Pull request closed without merging**